### PR TITLE
Clarify start steps

### PR DESCRIPTION
**Pull request by lilred**
For Bazel n00bs like me.

### CODE REVIEW

**Comment by lilred** [comment]
CLA signed

File: README.md [hunk 1]
```diff
@@ -71,10 +71,12 @@ you need:
 
   * [Install JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
   * [Install Bazel](https://docs.bazel.build/versions/master/install.html).
+  * Clone the copybara sorce locally: 
+      * `git clone git@github.com:google/copybara.git`
```
Comment (denian) [inline README.md]: Prefer

`git clone https://github.com/google/copybara.git`

as it doesn't require SSH keys setup.

**Comment by denian** [comment]
Thanks!

Culd you please squash the commits in this PR? We do require that to maintain a clean history.

**Comment by lilred** [comment]
OK, done

**Comment by denian** [comment]
> OK, done
> 
> (FYI you can use GitHub "squash and merge" for that, it's pretty great)

Not really in our case, our source-of-truth is an internal repo, so we need to import the change first, merge internally and then export and the pull request gets merged automatically.

Thanks

**Comment by denian** [comment]
Change was merged.

**Pull request closed without merging**